index.html

```diff
@@ -40,5 +40,157 @@ <h2>I'm a software developer</h2>
             </li>
         </ul>
     </section>
+    <section>
+        <div class="spacing">
+            <a class="disabled" href=""><img src="images/Disabled.png" alt="next"></a>
+            <h2 class="subtittle2">My Recent<br>Work</h2>
+        </div>
+        <div class="card">
+            <div class="box cards1">
+                <div class="blank">
+                    <div class="cardscontainers">
+                        <p class="cardtittle">Multi-Post Stories<br>Gain+Glory</p>
+                        <ul class="ul1">
+                            <li class="ruby2">Ruby on rails</li>
+                            <li class="css0">css</li>
+                            <li class="java0">JavScript</li>
+                            <li class="html0">html</li>
+                        </ul>
```

Comment:
Your *Used languages* section is looking pretty good, but kindly add a tiny bit of `border-radius` to them, also, try fixing this issue, where the items get outside the card :mag: 

![image](https://user-images.githubusercontent.com/78335447/170850729-2cabd72e-59f3-4293-b82d-a14d4c5b2599.png)